integration-tests/gherkin-rules.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@Bertk would move this file to
* https://github.com/SonarOpenCommunity/sonar-cxx/tree/master/integration-tests/features.
* add a comment at the beginning that this file is to use [SonarQube Gherkin plugin](https://github.com/racodond/sonar-gherkin-plugin)

Reply:
:ok: